druid-shell/src/backend/gtk/application.rs

```diff
@@ -84,7 +84,11 @@ impl Application {
     }
 
     pub fn get_locale() -> String {
-        glib::get_language_names()[0].as_str().into()
+        let mut locale: String = glib::get_language_names()[0].as_str().into();
```

Comment:
I think it's worth a comment explaining why this is done (locale having one format but type used expecting a different format).